modules/serverless-creating-channel-admin-web-console.adoc

```diff
@@ -6,18 +6,14 @@
 [id="serverless-creating-channel-admin-web-console_{context}"]
 = Creating a channel by using the Administrator perspective
 
-ifdef::openshift-enterprise[]
-If you have cluster administrator permissions, you can create a channel by using the *Administrator* perspective in the web console.
-endif::[]
-
-ifdef::openshift-dedicated[]
-If you have cluster or dedicated administrator permissions, you can create a channel by using the *Administrator* perspective in the web console.
-endif::[]
+include::snippets/serverless-channels-intro.adoc[]
```

Comment:
I guess this is the same for the develop view right?

Reply:
Yeah, it's just content reuse anywhere there's an "include" statement, you should be able to look at both in the preview link by navigating around the docs 🙂